code/ui/blocks/src/components/DocsPage.tsx

```diff
@@ -40,9 +63,374 @@ export const Subtitle = styled.h2(withReset, ({ theme }) => ({
   color: transparentize(0.25, theme.color.defaultText),
 }));
 
```

Comment:
The following style are copied from all the components in https://github.com/storybookjs/storybook/tree/tom/sb-996-resolve-css-issue-with-introduction/code/ui/components/src/typography/elements